RadarSDK/RadarLocationManager.m

```diff
@@ -518,7 +518,7 @@ - (void)handleLocation:(CLLocation *)location source:(RadarLocationSource)source
             if (duration == 0) {
                 duration = -[location.timestamp timeIntervalSinceNow];
             }
-            stopped = (distance <= options.stopDistance && duration >= options.stopDuration) || source == RadarLocationSourceVisitArrival;
+            stopped = distance <= options.stopDistance && duration >= options.stopDuration;
```

Comment:
should we add a tracking option `treatVistisAsStops` or just remove it like this? 🤔

Reply:
Considerations for leaving it:
- Although it can cause premature stops, in some cases it can also cause faster stops. Some people may want to leave it on for that reason.
- There could theoretically be some situations where location keeps bouncing around and the device never transitions to stopped, and a `CLVisit` is sort of a backstop for the OS to declare that the device is stopped.

That said, I'm inclined to keep it simple and remove it for now.

Reply:
I could make a tracking option defaulted to false pretty easily.

`useVisitsAsStops = false`?

Reply:
I think we can just change the behavior and skip adding a new tracking option. Suggest we merge this in with the rest of the iOS 3.2 beta stuff and do a bunch of testing in a variety of situations (urban and rural areas) in the next few weeks.